caikit/runtime/server_base.py

```diff
@@ -88,6 +96,23 @@ def _start_metrics_server(cls) -> None:
                 start_http_server(get_config().runtime.metrics.port)
             cls._metrics_server_started = True
 
+    @classmethod
+    def _get_runtime_info(cls) -> Dict[str, str]:
+        """Get information on versions of libraries and server from config"""
+        version_dict = {}
+        # TODO: how to get library version for extensions?
+        # version_dict["caikit_nlp_version"] = pip_version("caikit_nlp")
+        version_dict["caikit_version"] = pip_version("caikit")
```

Comment:
how do I get the library extension name without explicitly setting caikit_nlp since this should be generic for other extensions as well

Reply:
--> not possible to get `caikit_nlp` from within caikit, just can get caikit version. Is that sufficient/what is wanted?